platforms/documentation/docs/src/docs/userguide/dep-man/dependency-management/component_metadata_rules.adoc

```diff
@@ -78,153 +72,130 @@ include::sample[dir="snippets/dependencyManagement/customizingResolution-metadat
 ====
 
-In this case, it is pretty clear that the classifier stands for a target Java version, which is a <<variant_model.adoc#sec:variant-aware-matching,known Java ecosystem attribute>>.
-Because we also need both a _compile_ and _runtime_ for Java 8, we create two new variants but use the existing _compile_ and _runtime_ variants as _base_.
-This way, all other Java ecosystem attributes are already set correctly and all dependencies are carried over.
-Then we set the `TARGET_JVM_VERSION_ATTRIBUTE` to `8` for both variants, remove any existing file from the new variants with `removeAllFiles()`, and add the jdk8 jar file with `addFile()`.
-The `removeAllFiles()` is needed, because the reference to the main jar `quasar-core-0.7.5.jar` is copied from the corresponding base variant.
+In this case, the `jdk8` classifier clearly indicates the target Java version, which corresponds to a known attribute in the Java ecosystem.
+Since we need both compile and runtime variants for Java 8, we create two new variants using the existing compile and runtime variants as a base.
+This ensures that all other Java ecosystem attributes are set correctly, and dependencies are carried over.
+
+We assign the `TARGET_JVM_VERSION_ATTRIBUTE` to `8` for both new variants, remove any existing files with `removeAllFiles()`, and then add the `jdk8` jar using `addFile()`. Removing the files is necessary because the reference to the main jar `quasar-core-0.7.9.jar` is copied from the base variant.
 
-We also enrich the existing _compile_ and _runtime_ variants with the information that they target Java 7 — `attribute(TARGET_JVM_VERSION_ATTRIBUTE, 7)`.
+Finally, we enrich the existing compile and runtime variants with the information that they target Java 7 using `attribute(TARGET_JVM_VERSION_ATTRIBUTE, 7)`.
 
-Now, we can request a Java 8 versions for all of our dependencies on the compile classpath in the build script and Gradle will automatically select the best fitting variant for each library.
-In the case of Quasar this will now be the _jdk8Compile_ variant exposing the `quasar-core-0.7.9-jdk8.jar`.
+With these changes, you can now request Java 8 versions for all dependencies on the compile classpath, and Gradle will automatically select the best-fitting variant. In the case of Quasar, this will be the `jdk8Compile` variant, which exposes the `quasar-core-0.7.9-jdk8.jar`.
 
-.Applying and utilising rule for Quasar metadata
 ====
 include::sample[dir="snippets/dependencyManagement/customizingResolution-metadataRule/kotlin",files="build.gradle.kts[tags=quasar-dependencies]"]
 include::sample[dir="snippets/dependencyManagement/customizingResolution-metadataRule/groovy",files="build.gradle[tags=quasar-dependencies]"]
 ====
 
-== Making variants encoded in versions explicit  ==
+With this configuration, Gradle will select the Java 8 variant of Quasar for the compile classpath.
+
+== Making variants encoded in versions explicit
```

Comment:
"The disadvantage is that there is no information what the different version suffixes mean semantically." below should be:
"The disadvantage is that there is no information AS TO what the different version suffixes mean semantically."

Reply:
Fixed. TY.